.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       with:
         # NOTE: full version (including .0) to work around
         # <https://github.com/dtolnay/rust-toolchain/issues/112>.
-        toolchain: 1.70.0
+        toolchain: 1.71.0
```

Comment:
We're not going to update our MSRV for these dev-dependency updates.

Reply:
Well, then we can't update 🤷

Reply:
I don't want to bump MSRV without good reason. And windows-sys is not a good reason imo, they release way too often with breaking changes and bump the MSRV too often.